src/testcafe/schema.json

```diff
@@ -162,6 +162,26 @@
         }
       }
     },
+    "compilerOptions": {
+      "type": "object",
+      "description": "[TestCafe] Compiler options",
+      "properties": {
+        "typescript": {
```

Comment:
Note, that the `typescript` object contains not only `customCompilerModulePath` and `configPath` properties. It can accept all available [TypeScript options](https://www.typescriptlang.org/docs/handbook/compiler-options.html). For more information see the [Compiler options](https://testcafe.io/documentation/402659/reference/testcafe-api/runner/compileroptions?search#header) help topic. So, you need to change the schema.